client/src/Components/InterviewDialog.js

```diff
@@ -0,0 +1,88 @@
+import React, { useContext } from 'react';
+
+import { useStyles } from '../themes/theme';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Dialog from '@material-ui/core/Dialog';
+
+import { StartDashboardButton } from './Buttons';
+import InterviewDifficultyMenu from './InterviewDifficultyMenu';
+import Axios from 'axios';
+import { store } from '../context/store';
+import { useHistory } from 'react-router';
+
+function SimpleDialog({ onClose, selectedValue, open, handleChange }) {
+  const classes = useStyles();
+
+  const handleClose = () => {
+    onClose(selectedValue);
+  };
+
+  const { state } = useContext(store);
+  const history = useHistory();
+
+  const createInterview = async () => {
+    try {
+      const { data } = await Axios.post(
+        '/interviews',
+        { creator: state.user._id },
+        { withCredentials: true }
+      );
+
+      history.push(`/lobby/${data.interview._id}`);
+    } catch (err) {
+      console.error('OUTPUT: SimpleDialog -> err', err);
+    }
+  };
+
+  return (
+    <Dialog onClose={handleClose} aria-labelledby='simple-dialog-title' open={open}>
+      <div className={classes.createInterviewDialog}>
+        <DialogTitle id='simple-dialog-title' className={classes.pastPracticesText}>
+          Create
+        </DialogTitle>
+        <InterviewDifficultyMenu
+          selectedValue={selectedValue}
+          handleChange={handleChange}
+        />
+        <StartDashboardButton onClick={createInterview}>Create</StartDashboardButton>
+      </div>
+    </Dialog>
+  );
+}
+
+export default function SimpleDialogDemo() {
```

Comment:
I don't know if you need these two separate components unless you are using one of the dialogs more than once - it is nice to split into many components, but only if you have a good use for the reusability and I feel because of how these two components interact, they are not as reusable.